softwareComponents/configuration/rofibot.h

```diff
@@ -155,6 +170,10 @@ class Module {
             _rootComponent = _computeRoot();
     }
 
+    Module( Module&& o ) noexcept {
+        swap( o );
```

Comment:
swap může nechat alokované vektory. Je otázka jestli nám to vadí, ale je to nečekané chování.
Proč to není `Module( Module&& o ) noexcept = default;`? ... a když jsme u toho, proč definujeme copy constructor? Nevypadá, že by dělal něco jiného než `default`. A `swap` může být implementovaný jako `std::swap` (po tom, co dáme move constructor default).

Reply:
A teda - nemusíme dávat `= default` pokud se má chovat celkově defaultně - prostě to smaž.

Reply:
Souhlasím. Podle mě je to pozůstatek z minulosti.